src/http/interceptors.ts

```diff
@@ -45,9 +50,18 @@ export const camelizeResponseData = ({
     ['arraybuffer', 'stream'].includes(config.responseType ?? '') ||
     config?.interceptors?.skipCamelizeResponseData
       ? data
-      : camelizeKeys(data),
+      : camelizeKeys(data, config.skipKeyNormalizationForProperties || []),
```

Comment:
Are we sure we want to keep this name? I still find it hard to reason with a negative (`skip`) in the name? 
Can we call it `normalizeCustomProperties` and default to `false` ?

Reply:
You mean default it to the empty array? 🤔

Reply:
Comment is in the wrong place 😒  Going to put it in the correct place.